storage/core/src/common/Interfaces.ts

```diff
@@ -26,7 +28,7 @@ export interface ObjectReference extends ObjectDirectory {
   objectName: string;
 }
 
-export interface ObjectProperties {
+export type ObjectProperties = ContentHeaders & {
```

Comment:
`contentEncoding` in `ContentHeaders` is nullable. Should it be nullable in `ObjectProperties`? I would assume that there is always some kind of default value specified or is this a wrong assumption?

Reply:
it's nullable in ContentHeaders, and it should be undefined by default

Reply:
![image](https://user-images.githubusercontent.com/66480813/193062425-43a314f0-04fd-4597-ab81-943d64c824f9.png)
here's the definition of that property, it's nullable

Reply:
Ah I see, ok